bigquery/google/cloud/bigquery/magics.py

```diff
@@ -369,6 +403,17 @@ def _cell_magic(line, query):
     job_config = bigquery.job.QueryJobConfig()
     job_config.query_parameters = params
     job_config.use_legacy_sql = args.use_legacy_sql
+
+    if args.maximum_bytes_billed == "None":
+        job_config.maximum_bytes_billed = 0
+    elif args.maximum_bytes_billed is not None:
+        try:
+            value = int(args.maximum_bytes_billed)
+            job_config.maximum_bytes_billed = value
+        except Exception:
```

Comment:
Ditto. Don't catch `Exception`. We do need to think about having better error messages in our magics, but I don't think this is the way to do it.